### PR TITLE
updates flutter SDK overview links

### PR DESCRIPTION
**Pull request by ryanmcginnis**
- adds link to tools/dart-tool
- update description for tools/dart-vm

### CODE REVIEW

File: src/tools/sdk/index.md [hunk 1]
```diff
@@ -29,8 +29,11 @@ directory that has these command-line tools:
 
 <div class="row">
   <div class="col-lg-6" markdown="1">  
-    
-  [dart](/tools/dart-vm)
+
+  [dart](/tools/dart-tool)
+  : The Dart command line tool
+
+  [dart-vm](/tools/dart-vm)
```
Comment (kwalrath) [inline src/tools/sdk/index.md:36]: Good catch! Actually, why don't we just delete the entry for the standalone VM? People can get to it from the dart-tool page.

File: src/tools/sdk/index.md [hunk 2]
```diff
@@ -29,8 +29,11 @@ directory that has these command-line tools:
 
 <div class="row">
   <div class="col-lg-6" markdown="1">  
-    
-  [dart](/tools/dart-vm)
+
+  [dart](/tools/dart-tool)
+  : The Dart command line tool
```
Comment (kwalrath) [inline src/tools/sdk/index.md:34]:
```suggestion
  : The Dart command-line tool
```

File: src/tools/sdk/index.md (hunk 1 above)
Comment (ryanmcginnis) [reply src/tools/sdk/index.md]: done, i think. i haven't worked in Git for a while.

**Review by kwalrath — APPROVED** [review]
LGTM!

**Pull request merged**